models/api.models.js

```diff
@@ -28,24 +28,15 @@ exports.fetchArticles = () => {
 	});
 };
 
-let validArticleIds = [];
-
-exports.getValidArticleIds = () => {
-	let sqlQuery = `SELECT * FROM articles`;
-	return db.query(sqlQuery).then((result) => {
-        const articleArr = result.rows
-        articleArr.forEach((article) => {
-            validArticleIds.push(article.article_id)
-        })
-	});
-};
-
 exports.fetchArticleById = (id) => {
-	if (id > validArticleIds.length || isNaN(id)) {
-		return Promise.reject({ status: 400, msg: 'Bad Request' });
-	}
-	let sqlQuery = `SELECT * FROM articles WHERE article_id = $1`;
-	return db.query(sqlQuery, [id]).then((result) => {
-		return result.rows[0];
-	});
+    if (isNaN(id)) {
+        return Promise.reject({ status: 400, msg: 'Bad Request' });
+    }
```

Comment:
When we try make a psql query with an invalid datatype it will send an error back - usually we'd want to make use of that error - check out [PSLQ Thrown Errors](https://notes.northcoders.com/courses/js-back-end/error-handling) on the notes page for a reminder of what this looks like